{"uid":"c3e3ef4267036937","name":"test_disconnect_from_cluster_removes_user","fullName":"tests.integration.sharding_tests.test_mongos#test_disconnect_from_cluster_removes_user","historyId":"55a80986b29d75962efdc1d777664eb5","time":{"start":1728524903196,"stop":1728524932283,"duration":29087},"description":"Verifies that when the cluster is formed a the user is removed.","descriptionHtml":"<p>Verifies that when the cluster is formed a the user is removed.</p>\n","status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/tests/integration/sharding_tests/test_mongos.py\", line 161, in test_disconnect_from_cluster_removes_user\n    with pytest.raises(OperationFailure) as pymongo_error:\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 1011, in __exit__\n    fail(self.message)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 178, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728524522567,"stop":1728524522567,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728524522567,"stop":1728524522799,"duration":232},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728524522567,"stop":1728524522567,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728524903196,"stop":1728524903196,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Verifies that when the cluster is formed a the user is removed.","status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/tests/integration/sharding_tests/test_mongos.py\", line 161, in test_disconnect_from_cluster_removes_user\n    with pytest.raises(OperationFailure) as pymongo_error:\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 1011, in __exit__\n    fail(self.message)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 178, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","steps":[],"attachments":[{"uid":"fedf2b170a44d682","name":"log","source":"fedf2b170a44d682.txt","type":"text/plain","size":1871}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1728524932284,"stop":1728524932616,"duration":332},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1728524932284,"stop":1728524932284,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728524932616,"stop":1728524932616,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728524932616,"stop":1728524932617,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728524932617,"stop":1728524932617,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.sharding_tests"},{"name":"suite","value":"test_mongos"},{"name":"host","value":"fv-az730-517"},{"name":"thread","value":"13561-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.sharding_tests.test_mongos"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"6037d0ecdd345647","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":5,"broken":0,"skipped":20,"passed":12,"unknown":1,"total":38},"items":[{"uid":"4089e7d5609ff180","reportUrl":"../2514//#testresult/4089e7d5609ff180","status":"passed","time":{"start":1728438299196,"stop":1728438323997,"duration":24801}},{"uid":"c09ba619c609ecce","reportUrl":"../2493//#testresult/c09ba619c609ecce","status":"passed","time":{"start":1728351732451,"stop":1728351757292,"duration":24841}},{"uid":"c4bea153244b3da4","reportUrl":"../2484//#testresult/c4bea153244b3da4","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","time":{"start":1728265353551,"stop":1728265378334,"duration":24783}},{"uid":"888a33c5f90ffed1","reportUrl":"../2483//#testresult/888a33c5f90ffed1","status":"passed","time":{"start":1728178996372,"stop":1728179021189,"duration":24817}},{"uid":"5b3b1bf89fd625fa","reportUrl":"../2481//#testresult/5b3b1bf89fd625fa","status":"passed","time":{"start":1728006218950,"stop":1728006243704,"duration":24754}},{"uid":"4a3f90bf05d5938c","reportUrl":"../2477//#testresult/4a3f90bf05d5938c","status":"passed","time":{"start":1727920359817,"stop":1727920384662,"duration":24845}},{"uid":"6ba82e895391eb29","reportUrl":"../2465//#testresult/6ba82e895391eb29","status":"passed","time":{"start":1727833681783,"stop":1727833706618,"duration":24835}},{"uid":"41833a1a7a46084b","reportUrl":"../2456//#testresult/41833a1a7a46084b","status":"passed","time":{"start":1727746984746,"stop":1727747009595,"duration":24849}},{"uid":"d3c2dd0c36b3121c","reportUrl":"../2444//#testresult/d3c2dd0c36b3121c","status":"passed","time":{"start":1727660606112,"stop":1727660630990,"duration":24878}},{"uid":"d7fe879cafc38402","reportUrl":"../2443//#testresult/d7fe879cafc38402","status":"passed","time":{"start":1727574114592,"stop":1727574139467,"duration":24875}},{"uid":"475458aa66481c9b","reportUrl":"../2438//#testresult/475458aa66481c9b","status":"passed","time":{"start":1727401446113,"stop":1727401470891,"duration":24778}},{"uid":"3a2862b8e6a8ec3f","reportUrl":"../2434//#testresult/3a2862b8e6a8ec3f","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","time":{"start":1727314950795,"stop":1727314975635,"duration":24840}},{"uid":"8b9b21818546ca70","reportUrl":"../2429//#testresult/8b9b21818546ca70","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","time":{"start":1727228707814,"stop":1727228732691,"duration":24877}},{"uid":"29eee5d62befe295","reportUrl":"../2421//#testresult/29eee5d62befe295","status":"passed","time":{"start":1727142176008,"stop":1727142200823,"duration":24815}},{"uid":"f89f547a0db83353","reportUrl":"../2357//#testresult/f89f547a0db83353","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'pymongo.errors.OperationFailure'>","time":{"start":1726191571222,"stop":1726191596076,"duration":24854}},{"uid":"ead76d539ae33f5b","reportUrl":"../2348//#testresult/ead76d539ae33f5b","status":"passed","time":{"start":1726105186160,"stop":1726105211442,"duration":25282}},{"uid":"b25ba71b028730d4","reportUrl":"../2330//#testresult/b25ba71b028730d4","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726019381048,"stop":1726019381048,"duration":0}},{"uid":"7a977bb19a955f96","reportUrl":"../2317//#testresult/7a977bb19a955f96","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725846512498,"stop":1725846512498,"duration":0}},{"uid":"d909d8492e89c46d","reportUrl":"../2316//#testresult/d909d8492e89c46d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725760227995,"stop":1725760227995,"duration":0}},{"uid":"71446ea732f02562","reportUrl":"../2315//#testresult/71446ea732f02562","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725674218396,"stop":1725674218396,"duration":0}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"c3e3ef4267036937.json","parameterValues":[]}